kinto/views/__init__.py

```diff
@@ -29,4 +30,10 @@ def object_exists_or_404(request, collection_id, object_id, parent_id=''):
                            object_id=object_id)
     except exceptions.RecordNotFoundError:
         # XXX: We gave up putting details about parent id here (See #53).
-        raise httpexceptions.HTTPNotFound()
+        detail_dict = {
+                "id": collection_id,
+                "resource_name": "collection"
+            }
```

Comment:
I see where we define the custom 404 for collections, and for records, but I don't see any place where we define one for buckets. Where is that?